src/main/java/com/esri/samples/imagelayers/query_map_image_sublayer/QueryMapImageSublayerSample.java

```diff
@@ -0,0 +1,225 @@
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.layers.ArcGISMapImageLayer;
+import com.esri.arcgisruntime.layers.ArcGISMapImageSublayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+
+public class QueryMapImageSublayerSample extends Application {
+
+  private MapView mapView;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    try {
+      // create a border pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+      scene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+      // size the stage and add a title
+      stage.setTitle("Query Map Image Sublayer Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a map with a basemap and set its initial viewpoint
+      ArcGISMap map = new ArcGISMap(Basemap.createStreetsVector());
+      Point initialLocation = new Point(-12716000.00, 4170400.00, SpatialReferences.getWebMercator());
+      Viewpoint viewpoint = new Viewpoint(initialLocation, 6000000);
+      map.setInitialViewpoint(viewpoint);
+
+      // create and add a map image layer to the map
+      ArcGISMapImageLayer imageLayer = new ArcGISMapImageLayer("https://sampleserver6.arcgisonline.com/arcgis/rest/services/USA/MapServer");
+      map.getOperationalLayers().add(imageLayer);
+
+      // create a map view and set the map to it
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // create a graphics overlay to show the results in
+      GraphicsOverlay graphicsOverlay = new GraphicsOverlay();
+      mapView.getGraphicsOverlays().add(graphicsOverlay);
+
+      // create symbols for showing the results of each sublayer
+      SimpleMarkerSymbol citySymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, 0xFFFF0000, 16);
+      SimpleLineSymbol stateSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFF0000FF, 6);
+      SimpleLineSymbol countyLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DASH, 0xFF00FFFF, 2);
+      SimpleFillSymbol countySymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.DIAGONAL_CROSS, 0xFF00FFFF,
+          countyLineSymbol);
+
+      // create a vbox to hold the controls
+      VBox vBoxControl = new VBox(6);
+      vBoxControl.setMaxSize(160, 100);
```

Comment:
Would change size from 160 to 170 so that the whole text can be seen.
<img width="163" alt="screen shot 2018-06-04 at 2 51 12 pm" src="https://devtopia.esri.com/storage/user/1130/files/eefdf600-6806-11e8-9c61-89af8b9f54ef">